src/sorcha/readers/CombinedDataReader.py

```diff
@@ -128,10 +128,78 @@ def read_block(self, block_size=None, verbose=False, **kwargs):
             verboselog("Checking Object IDs in auxiliary data")
             current_ids = set(pd.unique(current_df["ObjID"]).astype(str))
             if not ephem_ids.issubset(current_ids):  # pragma: no cover
-                pplogger.error("ERROR: At least one missing ObjID in {reader.get_reader_info()}")
-                sys.exit("ERROR: At least one missing ObjID {reader.get_reader_info()}")
+                pplogger.error(f"ERROR: At least one missing ObjID in {reader.get_reader_info()}")
+                sys.exit(f"ERROR: At least one missing ObjID {reader.get_reader_info()}")
 
             verboselog("Joining auxiliary data with ephemeris")
             ephem_df = ephem_df.join(current_df.set_index("ObjID"), on="ObjID")
 
         return ephem_df
+
+    def read_aux_block(self, block_size=None, verbose=False, **kwargs):
```

Comment:
I don't particularly like this function - it seems like we should be able to use `read_block` instead of creating a new, almost identical version. @jeremykubica I would like to discuss this with you when you have a few minutes :)